ui/desktop/app/styles/app.scss

```diff
@@ -480,6 +480,10 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
   align-items: center;
   color: var(--ui-gray-subtler-2);
 
+  .rose-icon {
+    margin-right: sizing.rems(s);
```

Comment:
The secret eye icon isn't aligning vertically with secret hidden text. Can this be resolved with a margin-top?

Reply:
Now is perfectly aligned (measured and pixel perfect!)